crev-lib/src/util/mod.rs

```diff
@@ -159,7 +164,7 @@ pub fn get_recursive_digest_for_paths(
     root_path: &Path,
     paths: fnv::FnvHashSet<PathBuf>,
 ) -> std::result::Result<Vec<u8>, crev_recursive_digest::DigestError> {
-    let h = crev_recursive_digest::RecursiveDigest::<crev_common::Blake2b256, _, _>::new()
+    let mut h = crev_recursive_digest::RecursiveDigest::<crev_common::Blake2b256, _, _>::new()
```

Comment:
I think this `mut`s should be unnecessary now. You can try rebasing.

Reply:
That worked.